lib/pdk/validators/ruby/rubocop.rb

```diff
@@ -16,8 +16,12 @@ def self.cmd
         'rubocop'
       end
 
+      def self.pattern
+        '**/**.rb'
```

Comment:
Rubocop checks a number of different files e.g. .irbrc, Rakefile, Gemfile - should this pattern definitely restrict to *.rb ?
I think for a Puppet module we want `{Rakefile,Gemfile,**/**.rb}` (or whatever the glob syntax for _or_ is )